src/app.rs

```diff
@@ -47,15 +48,33 @@ impl App {
 
         github.scopes.push(String::from("read:org"));
 
+        let db_pool_size = match (env::var("DB_POOL_SIZE"), config.env) {
```

Comment:
Do you think it's worth just removing the `env` defaults and assume that if the var is unset it's in development mode?